README.md

```diff
@@ -79,18 +79,24 @@ kogitoapp.app.kiegroup.org/example-quarkus created
 oc delete kogitoapp example-quarkus
 ```
 
-## Development
+## Development 
 
-Change log level at runtime w/ the `DEBUG` environment variable. e.g. -
+Change log level at runtime with the `DEBUG` environment variable. e.g. -
 
 ```bash
 make dep
 make clean
 DEBUG="true" operator-sdk up local --namespace=<namespace>
 ```
 
```

Comment:
WDYT over add the steps to set up the local env to debug it locally?

Reply:
Added to my plate for future.